### PR TITLE
Icon font selector

### PR DESCRIPTION
**Pull request by pixelmatseriks**
copied userFunc IconFontSelector to theme_t3kit, replaced usage of t3kit_extension_tools IconFontSelector to theme_t3kit IconFontSelector.

### CODE REVIEW

File: Configuration/FlexForms/flexform_bigIconTextButton.xml [hunk 1]
```diff
@@ -32,8 +32,8 @@
                             <label>LLL:EXT:theme_t3kit/Resources/Private/Language/ContentElements.xlf:bigIconTextButton.flexform.iconClass</label>
                             <config>
                                 <type>user</type>
-                                <userFunc>T3kit\T3kitExtensionTools\UserFunction\IconFontSelector->renderField</userFunc>
-                                <cssFile>EXT:theme_t3kit/Resources/Public/felayout_t3kit/fonts/style.css</cssFile>
+                                <userFunc>T3kit\themeT3kit\UserFunction\IconFontSelector->renderField</userFunc>
+                                <cssFile>EXT:theme_t3kit/Resources/Public/felayout_t3kit/dev/fonts/style.css</cssFile>
```
Comment (MattiasNilsson) [inline Configuration/FlexForms/flexform_bigIconTextButton.xml:36]: I am not sure we should use the felayout_t3kit path at all?

File: Configuration/FlexForms/flexform_div.xml [hunk 2]
```diff
@@ -15,8 +15,8 @@
 							<label>LLL:EXT:theme_t3kit/Resources/Private/Language/ContentElements.xlf:iconTextButton.flexform.iconClass</label>
 							<config>
 								<type>user</type>
-								<userFunc>T3kit\T3kitExtensionTools\UserFunction\IconFontSelector->renderField</userFunc>
-								<cssFile>EXT:theme_t3kit/Resources/Public/felayout_t3kit/fonts/style.css</cssFile>
+								<userFunc>T3kit\themeT3kit\UserFunction\IconFontSelector->renderField</userFunc>
+								<cssFile>EXT:theme_t3kit/Resources/Public/felayout_t3kit/dev/fonts/style.css</cssFile>
```
Comment (MattiasNilsson) [inline Configuration/FlexForms/flexform_div.xml:19]: I am not sure we should use the felayout_t3kit path at all?

File: Configuration/FlexForms/flexform_iconTextButton.xml [hunk 3]
```diff
@@ -32,8 +32,8 @@
                             <label>LLL:EXT:theme_t3kit/Resources/Private/Language/ContentElements.xlf:iconTextButton.flexform.iconClass</label>
                             <config>
                                 <type>user</type>
-                                <userFunc>T3kit\T3kitExtensionTools\UserFunction\IconFontSelector->renderField</userFunc>
-                                <cssFile>EXT:theme_t3kit/Resources/Public/felayout_t3kit/fonts/style.css</cssFile>
+                                <userFunc>T3kit\themeT3kit\UserFunction\IconFontSelector->renderField</userFunc>
+                                <cssFile>EXT:theme_t3kit/Resources/Public/felayout_t3kit/dev/fonts/style.css</cssFile>
```
Comment (MattiasNilsson) [inline Configuration/FlexForms/flexform_iconTextButton.xml:36]: I am not sure we should use the felayout_t3kit path at all?

File: Configuration/TCA/Overrides/pages.php [hunk 4]
```diff
@@ -4,8 +4,8 @@
 call_user_func(function() {
     $GLOBALS['TCA']['pages']['columns']['tx_themes_icon']['config'] = array (
         'type' => 'user',
-        'userFunc' => 'T3kit\T3kitExtensionTools\UserFunction\IconFontSelector->renderField',
-        'cssFile' => 'EXT:theme_t3kit/Resources/Public/felayout_t3kit/fonts/style.css',
+        'userFunc' => 'T3kit\themeT3kit\UserFunction\IconFontSelector->renderField',
+        'cssFile' => 'EXT:theme_t3kit/Resources/Public/felayout_t3kit/dev/fonts/style.css',
```
Comment (MattiasNilsson) [inline Configuration/TCA/Overrides/pages.php:8]: I am not sure we should use the felayout_t3kit path at all?

**Review by MattiasNilsson — CHANGES_REQUESTED** [review]
It looks good but I am not sure about the felayout_path?

File: Configuration/FlexForms/flexform_bigIconTextButton.xml (hunk 1 above)
Comment (dmh) [reply Configuration/FlexForms/flexform_bigIconTextButton.xml:36]: Even if we are not using `felayout_t3kit` as a separate repo I would like to keep it as a name of the folder inside of `theme_t3kit`. Just to let everyone know that it is `felayout` as it was in previous versions, and all FE stuff is here.

File: Configuration/FlexForms/flexform_div.xml (hunk 2 above)
Comment (dmh) [reply Configuration/FlexForms/flexform_div.xml:19]: Even if we are not using felayout_t3kit as a separate repo I would like to keep it as a name of the folder inside of theme_t3kit. Just to let everyone know that it is felayout as it was in previous versions, and all FE stuff is here.

**Review by dmh — APPROVED** [review]

**Comment by dmh** [comment]
we will change felayout_t3kit path later after changing FE configurations

**Pull request merged**